djimaging/tables/response/orientation.py

```diff
@@ -93,11 +95,12 @@ def get_time_dir_kernels(sorted_responses, dt):
     dir_component *= s
 
     # determine which entries correspond to the first second, assuming 4 seconds presentation time
-    first_second_idx = np.maximum(int(np.round(1. / dt)), 1)
+    first_second_idx = np.maximum(int(np.floor(1.0 / dt)), 1)
```

Comment:
The original code hardcoded this to 7 frames. We were using 8 with round, so to keep the flexibility but mimic the original behaviour I changed this to floor

Reply:
ok